lib/src/Base/Stat/LinearModelFactory.cxx

```diff
@@ -18,91 +18,32 @@
  *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-#include <fstream>
-#include <cstdlib>
 
 #include "openturns/LinearModelFactory.hxx"
 #include "openturns/LinearModel.hxx"
-#include "openturns/Path.hxx"
-#include "openturns/Exception.hxx"
-#include "openturns/OTconfig.hxx"
-#include "openturns/Log.hxx"
-#include "openturns/Os.hxx"
+#include "openturns/LinearModelAlgorithm.hxx"
+#include "openturns/LinearModelAnalysis.hxx"
```

Comment:
NOOOO
From Base we could not rely on Uncertainty objects!

Reply:
Ok, moved